DataFormats/TrackReco/interface/TrackExtraBase.h

```diff
@@ -44,9 +49,6 @@ class TrackExtraBase
         return m_nHits;
     }
 
-    /// accessor to RecHits
-    auto recHits() const { return TrackingRecHitRange(recHitsBegin(), recHitsEnd()); }

```

Comment:
This was added recently by #25591
What's the reason to remove it?

Reply:
I didn't intend to - strange that it is missing, I though started from the master branch as of this morning.

Reply:
Added back.